EnglishFilter/sections/specific.txt

```diff
@@ -2118,6 +2118,7 @@ dafontfree.io##div[data-ai-tracking] a[target]
 ||nudeclap.com/nc/nc.js
 nudeclap.com##.style75
 nudeclap.com##a[target="_blank"][rel="nofollow noopener"]
+albumoftheyear.org#?#div:has(> .adHeading)
```

Comment:
It seems it is not required.

Reply:
there is a leftover ad container, but maybe `albumoftheyear.org##div[style="padding: 30px 0 60px 0; text-align: center;"]` is better selector

Reply:
Try this
```
albumoftheyear.org##.fullWidth > div.albumBlock ~ div[style^="padding:"]
```

Reply:
That doesn't seem to work on this page, for example, `https://www.albumoftheyear.org/2022/releases/july-07.php`

Reply:
Maybe just `##.albumBlock ~ div[style^="padding:"]` ?

Reply:
Yes, you can do that or use the previous rule, but with the correction.
```
albumoftheyear.org##.fullWidth div.albumBlock ~ div[style^="padding:"]
```